include/boost/json/basic_parser_impl.hpp

```diff
@@ -217,8 +217,10 @@ const char*
 basic_parser<Handler>::
 sentinel()
 {
+    // the "+1" ensures that the returned pointer is unique even if
+    // the given input buffer borders on this object
     return reinterpret_cast<
```

Comment:
`static_assert(sizeof(*this) > 1);`

Reply:
`class basic_parser` has lots of fields, and is always larger than 1 byte, unless this class gets refactored and all fields get removed, which isn't going to happen. I don't think this `static_assert` adds real value, it's trivial to see it's always true.

Reply:
Agreed, lol